do.py

```diff
@@ -859,27 +859,27 @@ def help():
   build-matlab            - Build Matlab module
   build-matlab-sms        - Build SMS-EMOA example in Matlab
   build-octave            - Build Matlab module in Octave
-  build-python            - Build Python modules
-  build-python2           - Build Python 2 modules
-  build-python3           - Build Python 3 modules
-  install-postprocessing  - Install postprocessing (user-locally)
+  build-python*           - Build Python modules
```

Comment:
The star is likely to be misunderstood by some users as part of the command which needs to be typed. The simple solution I could think of is to rather add `^*` at the end of the explanation, like
```
build-python*           - Build Python modules
```
-> 
```
build-python            - Build Python modules^*
```